scripts/local_testnet/README.md

```diff
@@ -20,7 +20,7 @@ Modify `vars.env` as desired.
 Start a local eth1 ganache server plus boot node along with `BN_COUNT`
 number of beacon nodes and `VC_COUNT` validator clients.
 
-The `start_local_testnet.sh` script takes three options `-v VC_COUNT`, `-d DEBUG_LEVEL` and `-h` for help.
+The `start_local_testnet.sh` script takes four options `-v VC_COUNT`, `-d DEBUG_LEVEL`, `-p` to enable private tx proposals and `-h` for help.
```

Comment:
```suggestion
The `start_local_testnet.sh` script takes four options `-v VC_COUNT`, `-d DEBUG_LEVEL`, `-p` to enable builder proposals and `-h` for help.
```

The "private tx" thing is a bit outdated these days, we tend to refer to this as "builder proposals". You could update the docs in `start_local_testnet.sh` as well if you like

Reply:
@michaelsproul fixed.